modules/apps/users-admin/users-admin-impl/src/main/java/com/liferay/users/admin/internal/search/spi/model/query/contributor/UserKeywordQueryContributor.java

```diff
@@ -41,20 +41,27 @@ public void contribute(
 			keywordQueryContributorHelper.getSearchContext();
 
 		_addHighlightFieldNames(searchContext);
-
```

Comment:
any reason for this remove?

Reply:
No, this was a typo on my part. 😓 I've fixed it.